### PR TITLE
docs: update messaging

### PR DESCRIPTION
**Pull request by zackbrown**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,279 @@
...
+    gradients: fromFigmaGradients,
+    shadows: fromFigmaShadows,
+    typography: fromFigmaTypography
+};
 
-## Diez Enterprise Edition
+```
 
-Diez Enterprise Edition includes the Diez DocsGen add-on — an additional compiler target that allows you to build any Diez project into up-to-the-moment design language docs.  Customizable, searchable, self-hostable. Perfect for enterprise design systems.
+Extracted tokens are intended to be _imported_ and _referenced_ by other files in your project — then updated at will be designers, and extracted again whenever you want.  As another application of "semantic tokens," this separation of concerns allows you to treat design files as versionable code assets.
```
Comment (taylorpoe) [inline README.md]: s/"be designers"/"by designers"

Comment (zackbrown) [reply README.md]: Thank you — just pushed a fix, and cleaned this up a bit on the way

File: README.md [hunk 2]
```diff
@@ -1,76 +1,279 @@
...
+ 
+We came up with the "codename Diez" during a [Haiku team summit in Patagonia](https://www.haikuforteams.com/blog/remote-team-summit).  Then we decided to make the "codename" the "real name."  And here we are.
+
+**Why TypeScript?**
+
+While design tokens are often expressed in JSON or YAML, TypeScript solves several problems faced when wrangling real-world design tokens:
+
+   - **Complexity:** As codebases grow and change, YAML and JSON get complex and hard to reason about.  With limited or no affordances for multiple files, references, or comments, YAML and JSON quickly snowball into complexity or "no one touch this or it will break everything!" 
+
+   - **Modularity & reuse:** Maintainers of design tokens often find a need to reuse certain chunks of tokens across files.  A common example: a color palette may need to be referenced both in a typography definition and in a panel style — or in both a dark-mode and a light-mode theme.  In these cases, yaml and json leave authors holding the bag — either contrive a module system for JSON, or copy, paste, and pray no one needs to update this again.
```
Comment (taylorpoe) [inline README.md]: Take it or leave it: consistent capitalization (json, yaml)

**Comment by taylorpoe** [comment]
LGTM 💯

File: README.md [hunk 3]
```diff
@@ -1,76 +1,279 @@
...
+  foreground: rawColors.tiger,
+  foreground50: rawColors.tiger.lighten(.5),
+  background: rawColors.clover,
+  background50: rawColors.clover.lighten(.5),
+}
+```
+
+### Nesting & reuse example
+
+This slightly more complex example shows how multiple prefabs (in this case `Typograph`, `Color`, and the primitive type `number`) naturally compose to express complex hierarchies of data — without needing to maintain multiple declarations of the same data.
```
Comment (funnypapaya) [inline README.md]: "Typograph" and "Color" are written with capital letters and "number" isn't.

**Pull request merged**